src/runner.js

```diff
@@ -8,6 +8,8 @@ const {
 } = require('./utils');
 
 const check = (license) => {
+  if (!license) throw new Error('Error: You must provide a license to check.');
```

Comment:
In fact, yargs does it and the execution doesn't reach this part. But I wanted to make sure to make it independent from the cli just in case we want to run it in a different way in the future (e.g. GitHub action)